### PR TITLE
Straight package-manager integration

### PR DESCRIPTION
**Pull request by leotaku**
This has been discussed in #33 

Are the `README` entries generated, or should I just manually copy the `cort-deftest`s into the file?

### CODE REVIEW

**Comment by conao3** [comment]
Thanks!
README doesn't generate. please copy manually.

File: leaf-keywords-tests.el [hunk 1]
```diff
@@ -332,6 +332,84 @@ Example
...
+         '(progn
+            (straight-use-package 'leaf)
+            (straight-use-package 'leaf-polyfill)))
+       (leaf-pre-init)
+       (leaf-init)))
+
+    ((leaf leaf
+       :init (leaf-pre-init)
+       :straight (zenburn-theme :type git :host github
+                                :repo "fake/fake")
```
Comment (conao3) [inline leaf-keywords-tests.el]: please remove newline to connect line `line:386` and `line:387`

File: leaf-keywords-tests.el [hunk 2]
```diff
@@ -332,6 +332,84 @@ Example
...
+       :config (leaf-init))
+     (prog1 'leaf
+       (eval-after-load 'straight
+         '(progn
+            (straight-use-package '(zenburn-theme :type git :host github
+                                                  :repo "fake/fake"))))
```
Comment (conao3) [inline leaf-keywords-tests.el]: please remove newline to connect line `line:392` and `line:393`

**Comment by conao3** [comment]
Please check, and fix readme too. Sorry for your double work.

**Comment by conao3** [comment]
OK, thanks!
As proof that this PR is over, please increase the version of leaf-keywords to `v1.1.6` and add your name to [Contributors section](https://github.com/conao3/leaf-keywords.el#contributors) in the README.

**Comment by conao3** [comment]
Thank you for PR! Congrats on your first contribution! :tada:

**Comment by leotaku** [comment]
Great! Thanks for allowing this feature to be merged!

**Pull request merged**